swap/swap.go

```diff
@@ -461,6 +469,37 @@ func (s *Swap) Balances() (map[enode.ID]int64, error) {
 	return balances, nil
 }
 
+// AvailableBalance returns the total balance of the chequebook against which new cheques can be written
+func (s *Swap) AvailableBalance() (uint64, error) {
+	// get the LiquidBalance of the chequebook
+	liquidBalance, err := s.contract.LiquidBalance(nil)
+	if err != nil {
+		return 0, err
+	}
+
+	// get all sent Cheques
+	sentCheques, err := s.SentCheques()
```

Comment:
this is going to conflict with PR #1863, which removes `SentCheques`.

it can still be adapted though

Reply:
I will leave it now as is, as there is no alternative RPC call available in this branch. 
We can resolve any potential conflicts when we merge :)

Reply:
sure. just a heads up :smile: